bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -25,9 +25,6 @@
   {% endif %}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
Thinking I will create a new bug to tackle removing template-resp.css @alexgibson, @jpetto. For now I will update to page_css.